app/lib/dfe/reference_data/degrees.rb

```diff
@@ -1,730 +1,669 @@
+          '98db840e-a399-40c5-a95d-589e54ccee4c' => { 'name' => 'level 3 diploma', synonyms => [], level => '3' },
+          '0ab5bb24-3cde-4489-96ff-fd558c046474' => { 'name' => 'level 3 English for speakers of other languages', synonyms => ['ESOL'], level => '3' },
+          '537433b3-562e-4cac-b61e-b9386a80ace0' => { 'name' => 'level 3 national certificate', synonyms => [], level => '3' },
+          '67bc4f81-186a-4b50-a09d-4b50d64c4c0a' => { 'name' => 'level 3 national diploma', synonyms => [], level => '3' },
+          '1b2063df-13b3-4e46-8020-916708493273' => { 'name' => 'level 3 national vocational qualification', synonyms => ['NVQ'], level => '3' },
+          '1eb92c31-cd60-4ff6-8d0f-8dfde9705191' => { 'name' => 'music grades 6, 7 and 8', synonyms => [], level => '3' },
+          'a42468af-a30e-4893-852e-fa8aad0047a5' => { 'name' => 'tech level', synonyms => [], level => '3' },
+          'f7ab37d2-4413-41e6-b0aa-9b0cb7bf9d53' => { 'name' => 'certificate of higher education', synonyms => ['CertHE'], level => '4' },
+          'dbe7bb34-2df5-413a-8ce9-9b806da829e1' => { 'name' => 'higher apprenticeship', synonyms => [], level => '4' },
+          '8f08ac5c-5657-4616-ae97-146b5e6f5b67' => { 'name' => 'higher national certificate', synonyms => ['HNC'], level => '4' },
+          'aca93aaf-747b-489e-a76f-f25310e6f87d' => { 'name' => 'level 4 award', synonyms => [], level => '4' },
+          'de7c4d54-68bd-44c3-a8fe-70a9ff1847a4' => { 'name' => 'level 4 certificate', synonyms => [], level => '4' },
+          '1eb3625e-d376-4b0b-8521-8a1269ef73ac' => { 'name' => 'level 4 diploma', synonyms => [], level => '4' },
+          'b6ba9cfe-5336-4589-a466-674dded38563' => { 'name' => 'level 4 national vocational qualification', synonyms => ['NVQ'], level => '4' },
+          '50c8bfd7-8b08-48d8-ab6d-b6be9c17ff79' => { 'name' => 'diploma of higher education', synonyms => ['DipHE'], level => '5' },
+          '8b5bb841-c7fd-43e3-87d8-dc9653c54dda' => { 'name' => 'foundation degree', synonyms => [], level => '5' },
+          '2e4d3d83-bbf7-4d37-b451-0f6bbf3ac32a' => { 'name' => 'higher national diploma', synonyms => ['HND'], level => '5' },
+          'ab446a38-71d5-45c2-8ad6-23990dc66f16' => { 'name' => 'level 5 award', synonyms => [], level => '5' },
+          '679215a9-6b4f-41b2-b7d9-f179c3f241ed' => { 'name' => 'level 5 certificate', synonyms => [], level => '5' },
+          '2810949c-b17f-4b08-8461-f4e3775e81f3' => { 'name' => 'level 5 diploma', synonyms => [], level => '5' },
+          'fdd44a44-ce01-4a4c-bc99-1e17e69ad8fe' => { 'name' => 'level 5 national vocational qualification', synonyms => ['NVQ'], level => '5' },
+          '77c6065f-de57-4094-b588-c1317b0f0b3e' => { 'name' => 'degree apprenticeship', synonyms => [], level => '6' },
+          'b580a760-da23-4d38-b803-62ae11de6a65' => { 'name' => 'ordinary degree', synonyms => ['bachelors degree', "bachelor's degree"], level => '6' },
```

Comment:
"Bachelor degree" is more commonly used than "ordinary degree" (which sometimes means a bachelor degree with no honours):
```suggestion
          'b580a760-da23-4d38-b803-62ae11de6a65' => { 'name' => 'bachelor degree', synonyms => ['bachelors degree', "bachelor's degree", 'ordinary degree'], level => '6' },
```

Reply:
Done